index.html

```diff
@@ -0,0 +1,343 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="styles.css">
+        <title>Amjad's Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
please read more about meta tag and use it correctly here

Reply:
ok i will add 
- description
- keywords
- author
is there any other specific meta data that i need to use in the case of this portfolio ?

Reply:
fixed, please check